docs/cloud/guides/demo-project.mdx

```diff
@@ -0,0 +1,308 @@
+    </Info>
+  </Step>
+
+  <Step title="Add the API Key to Your Git Provider">
+    <Tabs
+      groupId="git-provider"
+      values={[
+        { label: "GitHub", value: "github" },
+        { label: "GitLab", value: "gitlab" },
+        { label: "Bitbucket", value: "bitbucket" },
+      ]}
+    >
+      <TabItem value="github">
+        1. Go to your GitHub repository
+        2. Navigate to **Settings** → **Secrets and variables** → **Actions**
+        3. Click **New repository secret**
+        4. Set the name to `WIDGETBOOK_CLOUD_API_KEY`
+        5. Set the value to your API key from Widgetbook Cloud
+        6. Click **Add secret**
+      </TabItem>
+      
+      <TabItem value="gitlab">
+        1. Go to your GitLab repository
+        2. Navigate to **Settings** → **CI/CD**
+        3. Expand the **Variables** section
+        4. Click **Add variable**
+        5. Set the key to `WIDGETBOOK_CLOUD_API_KEY`
+        6. Set the value to your API key from Widgetbook Cloud
+        7. Make sure **Protect variable** is checked if you want to restrict it to protected branches
+        8. Click **Add variable**
+      </TabItem>
+      
+      <TabItem value="bitbucket">
+        1. Go to your Bitbucket repository
+        2. Navigate to **Repository settings** → **Pipelines** → **Repository variables**
+        3. Click **Add**
+        4. Set the name to `WIDGETBOOK_CLOUD_API_KEY`
+        5. Set the value to your API key from Widgetbook Cloud
+        6. Make sure **Secured** is checked
+        7. Click **Add**
+      </TabItem>
+    </Tabs>
+  </Step>
+
+  <Step title="Re-run the CI/CD Pipeline">
+    <Tabs
+      groupId="git-provider"
+      values={[
+        { label: "GitHub", value: "github" },
+        { label: "GitLab", value: "gitlab" },
+        { label: "Bitbucket", value: "bitbucket" },
+      ]}
+    >
+      <TabItem value="github">
+        1. Go to your GitHub repository
+        2. Navigate to **Actions** tab
+        3. Enable GitHub Actions for the forked repository
+        4. Go to the Widgetbook Cloud Action and run it. 
+        5. Wait for the workflow to complete
+      </TabItem>
+      
+      <TabItem value="gitlab">
+        1. Go to your GitLab repository
+        2. Navigate to **Build** → **Pipelines**
+        3. Find the most recent pipeline
+        4. Click the **Retry** button (circular arrow icon)
+        5. Wait for the pipeline to complete
+      </TabItem>
+      
+      <TabItem value="bitbucket">
+        1. Go to your Bitbucket repository
+        2. Navigate to **Pipelines**
+        3. Find the most recent pipeline
+        4. Click the **Rerun** button
+        5. Wait for the pipeline to complete
+      </TabItem>
+    </Tabs>
+    
+    <Info>
+      After the pipeline completes successfully, you should see a new build appear in your Widgetbook Cloud project's builds section.
+    </Info>
+  </Step>
+
+  <Step title="Create a Component Change and Pull Request">
```

Comment:
In step 1, the cloning was only done for GitHub, and not for the other 2 providers.